### PR TITLE
[REVIEW] Add missing entries in `update-version.sh`

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR adds `raft-dask` & `pylibraft` to `update-version.sh`

### CODE REVIEW

File: ci/release/update-version.sh [hunk 1]
```diff
@@ -60,6 +60,8 @@ for FILE in conda/environments/*.yml; do
    sed_runner "s/rmm=${CURRENT_SHORT_TAG}/rmm=${NEXT_SHORT_TAG}/g" ${FILE};
    sed_runner "s/libraft-headers=${CURRENT_SHORT_TAG}/libraft-headers=${NEXT_SHORT_TAG}/g" ${FILE};
    sed_runner "s/pyraft=${CURRENT_SHORT_TAG}/pyraft=${NEXT_SHORT_TAG}/g" ${FILE};
+   sed_runner "s/raft-dask=${CURRENT_SHORT_TAG}/libraft-headers=${NEXT_SHORT_TAG}/g" ${FILE};
```
Comment (raydouglass) [inline ci/release/update-version.sh:63]: Are you intentionally rewriting `raft-dask` to `libraft-headers` or is that a copy-paste error?

File: ci/release/update-version.sh [hunk 2]
```diff
@@ -60,6 +60,8 @@ for FILE in conda/environments/*.yml; do
...
+   sed_runner "s/pylibraft=${CURRENT_SHORT_TAG}/libraft-headers=${NEXT_SHORT_TAG}/g" ${FILE};
```
Comment (galipremsagar) [inline ci/release/update-version.sh:64]:
```suggestion
   sed_runner "s/raft-dask=${CURRENT_SHORT_TAG}/raft-dask=${NEXT_SHORT_TAG}/g" ${FILE};
   sed_runner "s/pylibraft=${CURRENT_SHORT_TAG}/pylibraft=${NEXT_SHORT_TAG}/g" ${FILE};
```

File: ci/release/update-version.sh (hunk 1 above)
Comment (galipremsagar) [reply ci/release/update-version.sh]: Thanks for the catch, it was a copy-paste error.

**Review by raydouglass — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**